README.md

```diff
@@ -75,6 +75,12 @@ make run
 If everything goes well, Asterinas is now up and running inside a VM.
 If things are running slow look for `ENABLE_KVM` in the output it should be 1 to make sure KVM was actually turned on.
 
+Developer docs for LDOS's Asterinas are the [advanced instructions](docs/src/kernel/advanced-instructions.md). They include:
+- [x] userspace testing
+- [x] kernelspace testing
+- [ ] dev environment setup
+- [x] gdb setup information
```

Comment:
Why are these check boxes?

Reply:
The dev environment setup is missing.

Reply:
Ideally put a stub in it's place and put a `TODO:`. Having check boxes here makes no sense anyway because the reader doesn't know what they mean. I don't know if they render with mdbook either.